package.json

```diff
@@ -235,6 +235,14 @@
       "lcov",
       "text-summary"
     ],
+    "coverageThreshold": {
```

Comment:
Which tool(s) consumes this object?

Reply:
jest: https://jestjs.io/docs/en/configuration#coveragethreshold-object